stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -1240,3 +1240,57 @@ func.func @sort_bounds_and_unknown_rank(%input0: tensor<*xf32>, %input1: tensor<
   %2 = "hlo_test_infer.get_return_types"(%0) : (tensor<*xf32>) -> tensor<*xindex>
   func.return
 }
+
+// -----
+
+// CHECK-LABEL: func @reduce_window_bound
+func.func @reduce_window_bound(%arg0: tensor<?x?xf32, #stablehlo.type_extensions<bounds = [4, 2]>>,
+    %arg1: tensor<?x?xi32, #stablehlo.type_extensions<bounds = [5, 3]>>,
```

Comment:
Do we need to have both %arg0 and %arg1? It sounds like just one of them would be sufficient to test the new functionality introduced in this PR?